pkg/indexerlookup/options.go

```diff
@@ -100,3 +101,12 @@ func WithIpfsDhtCascade(b bool) Option {
 		return nil
 	}
 }
+
+// WithLegacyCascade sets weather to cascade finds legacy providers connecting only over bitswap
+// Enabled by default if unspecified.
```

Comment:
Is it enabled by default? We need it set [here](https://github.com/filecoin-project/lassie/pull/166/files#diff-5679bcb27bf3098cbfcea743d9c030ae3f2b5eb1ba3c9bfacb43326ab5e59f94R29) for that I think.

Reply:
damn I thought I did that but clearly I didn't

Reply:
ok fixed